benchmarks/run-benchmark.sh

```diff
@@ -3,12 +3,47 @@
 # Runs a benchmark using drun. The script assumes drun is available on the caller's path.
 set -euo pipefail
 
+# Remove downloaded didc, drun.
+trap 'set +e && rm drun didc && set -e' EXIT SIGINT
```

Comment:
Why delete drun and didc? Don't want to cache them to continue using them in the next runs?